### PR TITLE
Show amounts of completed questions and correct questions

### PR DESCRIPTION
**Pull request by Veronika121**

### CODE REVIEW

File: src/views/show-amounts.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { quizData } from '../data.js';
+
+export function createCounts() {
```
Comment (martijnarts) [inline src/views/show-amounts.js]: Pass in `quizData` as a parameter instead

File: src/listeners/onload.js [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
-    renderQuestions(ui, list);
+  renderQuestions(ui, list);
+});
+
+import { createCounts } from '../views/show-amounts.js';
+window.addEventListener('DOMContentLoaded', () => {
+  const ui = document.getElementById('user-interface');
+  const count = createCounts();
+
+  renderQuestions(ui, count);
```
Comment (martijnarts) [inline src/listeners/onload.js]: Move all of this into a function in handlers/ and then call all the needed handlers without parameters.

**Comment by Veronika121** [comment]
I've made changes. Is it fine now?

**Comment by martijnarts** [comment]
:shipit:

**Comment by martijnarts** [comment]
Oh I noticed you put some things into `sandbox.html`. I deleted that on the master branch, and am using `index.html` instead :)

**Comment by Veronika121** [comment]
Yes, first I thought I need it. But I don't need it.

**Review by martijnarts — APPROVED** [review]

**Pull request merged**